projects/legacy/components/input-date-range/input-date-range.component.ts

```diff
@@ -86,6 +87,7 @@ export class TuiInputDateRangeComponent
     private readonly mobileCalendar = inject(TUI_MOBILE_CALENDAR, {optional: true});
     private readonly options = inject(TUI_INPUT_DATE_OPTIONS);
     private readonly textfieldSize = inject(TUI_TEXTFIELD_SIZE);
+    private readonly nativeValue = signal(this.nativeFocusableElement?.value || '');
```

Comment:
It will always be `''`, because it's a view child, it doesn't exist in constructor. Are we sure it works right when there is a value in the input initially?

Reply:
fixed